File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void pr(void)
{
    printf("#");
}

void printHashtag(int i)
{
    for(int j = 1; j < i+1 ;i++)
    {
        pr();
    }
}

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while(height  < 1 || height > 8);

    for(int i = 1; i <= height; i++)
    {
        printHashtag(i);
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


def main():
    n = get_height()
    draw(n)

def get_height():
    while True:
        height = get_int("Height: ")
        if height in range(1, 9):
            break
    return height



def draw(height):
    # draw(height - 1)
    # for i in range(height):
    print(' ' * 0, end='')
    print('#' * height, end='')
    print(' ' * 2, end='')
    print('#' * height)


main()